keyboards/lotus58/keymaps/default/rules.mk

```
@@ -0,0 +1,9 @@
+
+OLED_DRIVER_ENABLE = yes
+ENCODER_ENABLE = yes
+CONSOLE_ENABLE = yes
+RGBLIGHT_ENABLE = yes
+EXTRAKEY_ENABLE = yes
+SPLIT_KEYBOARD = yes
+WPM_ENABLE = yes
+EXTRAFLAGS += -flto
```

Most of this should be enabled at the keyboard level. 

```suggestion
OLED_DRIVER_ENABLE = yes
WPM_ENABLE = yes
LTO_ENABLE = yes
```